tests/test_modeling_flax_bert.py

```diff
@@ -29,6 +29,7 @@
         FlaxBertForNextSentencePrediction,
         FlaxBertForPreTraining,
         FlaxBertForQuestionAnswering,
+        FlaxBertForSequenceClassification,
```

Comment:
Nice catch!

Reply:
Not mine, it's the script. I'll let it know you liked ;-)